docs/csharp/tour-of-csharp/arrays.md

```diff
@@ -0,0 +1,82 @@
+---
+title: C# Arrays | A tour of the C# language
```

Comment:
We want a convention. Is C# too broad? When these topics show up in search results, they will (in time) show up near results for more in depth coverage of the topic. Would developers be able to distinguish the overview from the in-depth topic? How much does that matter?

/cc @cartermp